### PR TITLE
test(settings): remove TestBed from settings effects specs

### PR DESCRIPTION
**Pull request by zetsnotdead**
Refactored settings effects specs as discussed in #389

### CODE REVIEW

File: src/app/settings/settings.effects.spec.ts [hunk 1]
```diff
@@ -73,16 +63,4 @@ describe('SettingsEffects', () => {
       );
     });
   });
-
-  it('should not call methods on AnimationsService and LocalStorageService for other actions', () => {
```
Comment (timdeschryver) [inline src/app/settings/settings.effects.spec.ts]: I think this test is removed by accident? Or was this intended?

Comment (tomastrajan) [reply src/app/settings/settings.effects.spec.ts]: @zetsnotdead pls have a look :)

Comment (zetsnotdead) [reply src/app/settings/settings.effects.spec.ts]: I actually deleted it deliberately. My reason being two fold:
- no other effect spec tests for actions that do nothing, so this test seemed a little out of place
- if we tests for all actions that trigger an event, can't we just assume that all other actions are a noop

If this reasoning does not seem right I will revert it.

Comment (timdeschryver) [reply src/app/settings/settings.effects.spec.ts]: I think I have the same mindset as @zetsnotdead here 😄 
Especially on the 2nd point.

Comment (tomastrajan) [reply src/app/settings/settings.effects.spec.ts]: @zetsnotdead @timdeschryver  thanks for clarification!

**Pull request merged**